docs/concepts/artifacts.md

```diff
@@ -25,7 +25,7 @@ Creating artifacts allows you to publish data from task and flow runs or outside
 
 ### Creating Link Artifacts
 
-To create a link artifact, use the `create_link_artifact()` function.
+To create a link artifact, use the `create_link_artifact()` function. For artifacts to appear in the Artifacts page of the Prefect UI, provide a name to `create_link_artifact()` in the `key` field. Without a `key`, artifacts will only be visible in the artifacts tab of the associated flow run or task run. Keys allow you to view the history of your artifact over time.
```

Comment:
Provide a "name" in the "key" field? Should we just say provide a "key"?

Instead of "Keys allow you to..." I would say "Providing a key allows you to..."

Reply:
I guess I'm a little concerned if users will know what a key is right off the bat.

Reply:
"Provide a key, a name that uniquely identifies a particular artifact, to the `create_link_artifact()` function to track and view its history. Without a `key`, the artifact will only be visible in the artifacts tab of the associated flow run or task run."

Reply:
Riffing off of that:

"To create a link artifact, use the `create_link_artifact()` function. To create multiple versions of the same artifact and see them grouped together in the Artifacts page of the Prefect UI, provide a `key` argument to the `create_link_artifact()` function to track and view an artifact's history over time. Without a `key`, the artifact will only be visible in the artifacts tab of the associated flow run or task run."

Reply:
I like that